md/pages.md

```diff
@@ -95,13 +95,15 @@ out.write("""<iframe src=../portal.js/#/admin/monitoring/${idProcess}-${idCase}?
 
 While you are developing a custom page, you can enable custom page debug mode for your portal. In debug mode, you can see changes to your custom page without importing a new zip archive.
 
+::: Before use setup tool for the studio context, you need to edit the file : `database.properties`, and re-define ``h2.database.dir=../../<yourRepository>/h2_database`  :::
```

Comment:
1. This is very weird to talk about setup-tool here
2. Be careful, this is not the right syntax for `:::`
3. you have extra **`**